server/homebrew.api.js

```diff
@@ -478,14 +493,42 @@ const api = {
 		}
 
 		res.status(204).send();
+	},
+	returnBrew : async (req, res)=>{
+		return res.status(200).json(req.brew);
+	},
+	shareAuthorChecks : async (req, res, next)=>{
+		// If logged in user is not an author:
+		if(!req.brew.authors.includes(req?.account?.username)){
+			// Remove Edit ID
+			req.brew.editId = undefined;
+			// Increase view count and lastViewed property
+			await HomebrewModel.increaseView({ shareId: req.params.id });
```

Comment:
Just thinking out loud:

This no longer handles the case to increase view count for unstubbed Google-only brews. I suppose this makes sense; view count/lastViewed only really matters in the context of the HB environment itself, and is not required to "rebuild" the document if HB goes down.

If `GoogleActions.increaseView()` is no longer being used anywhere, let's remove it or put a note somewhere to remove it later.

Reply:
Good point - I thought that this had already been changed, but on checking now I see that `GoogleActions.increaseView` is called in the current version of the SharePage. However, is the view count something that should be stored outside of the brew stub, now that they exist?
I'm tempted to just replicate the functionality and leave the philosophical questions for another day.